index.html

```diff
@@ -3,15 +3,39 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Alecxis's Portfolio</title>
+    <style>
+        body {background-color: powderblue;}
+        h1 {color: rgb(5, 9, 223);}
+        p {color: blue;}
+      </style>
 </head>
 <body>
-    <h1>Alecxis</h1>
+<section id="About">  
+    <h1>Alecxiskelly.com</h1>
     <h2>About</h2>
-    <p>I am a creative Tech/Entrepreneur from Philadelhia, now I am worldwide.</p>
+    <p>I am a creative Tech/Entrepreneur from Philadelphia, now I am worldwide.</p>
+</section>
     <h2>Experience</h2>
```

Comment:
This should be wrapped in a <section></section> tag just how the about section is laid out!